models/cow_protocol/ethereum/cow_protocol_ethereum_app_data.sql

```diff
@@ -1,5 +1,5 @@
 {{ config(alias='app_data',
-        post_hook='{{ expose_spells(\'["ethereum"]\',
+        post_hook='{{ expose_spells_hide_trino(\'["ethereum"]\',
```

Comment:
isn't this already done [here](https://github.com/duneanalytics/spellbook/pull/2325/files#diff-451a048785db006e756572785fc88c4d7cb914a9830e4fe3c6c96f9655c2f576)?

Reply:
sigh